### ISSUE TITLE
Erro in documentation?

### ISSUE THREAD

**Issue by jishi** [PatrickE94/pycalima#22]
`Clone this repo and then user run.py script to set the basic settings for your Calima. They will be applied if Calima was power cycled. Afterwards it will just poll Calima and output the data. You can pipe it to your home autmation scripts or or cronjob it to a file.`

I don't understand what this is referring to? 

I managed to build my own invocation but what you are describing sounds like something useful... but where is it?

**Comment by PatrickE94** [comment]
Yes, this is a remnant left in the documentation. The file it refers to was a script by another user. 

https://github.com/bortek/pycalima/blob/72d2311d2ce712ac3bb47849177fd3ae69eb1c8c/run.py

It essentially just configured his device to his liking and then polled for data. Since this manages his personal setting, and I consider this more of a library. I eventually removed it. But please, feel inspired to use it!
